src/topology/algebra/matrix.lean

```diff
@@ -0,0 +1,40 @@
+/-
+Copyright (c) 2021 Oliver Nash. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Oliver Nash
+-/
+import linear_algebra.determinant
+import topology.algebra.ring
+
+/-!
+# Topological properties of matrices
+
+This file is a place to collect topological results about matrices.
+
+## Main definitions:
+
+ * `continuous_det`: the determinant is continuous over a topological ring.
+-/
+
+open matrix
+
+variables (k : Type*) [comm_ring k] [topological_space k] [topological_ring k]
+
+instance {ι : Type*} : topological_space (matrix ι ι k) := Pi.topological_space
+
+lemma continuous_det {n : ℕ} : continuous (det : matrix (fin n) (fin n) k → k) :=
```

Comment:
Can you add the version for other finite index types?

Reply:
Absolutely! Small chance I won't have time for a week (about to head on holidays) but will try to get it done shortly.

Reply:
Phew, I managed to get this done in my last few minutes.